.devcontainer/devcontainer.json

```
@@ -54,7 +56,11 @@
         // Allow the container to access the host X11 display and EPICS CA
         "--net=host",
         // Make sure SELinux does not disable with access to host filesystems like tmp
-        "--security-opt=label=disable"
+        "--security-opt=label=disable",
+        // Mount the user sockets folder
+        "-v${localEnv:XDG_RUNTIME_DIR}:${localEnv:XDG_RUNTIME_DIR}",
+        // add the docker socket environment variable to the container
+        "-e=DOCKER_HOST=${localEnv:DOCKER_HOST}"
```

This requires the following to be run:
https://github.com/gilesknap/docker-outside-docker/tree/main?tab=readme-ov-file#rootless-podman-support

The question is, do we want to document this, or run it for the user in `initializeCommand`? If the second, then we can add `systemctl --user enable podman --now` to `initializeCommand`, then this line can become:
```suggestion
        "-e=DOCKER_HOST=${localEnv:XDG_RUNTIME_DIR}/podman/podman.sock"
```